.swiftlint.yml

```diff
@@ -39,7 +39,6 @@ whitelist_rules:
   - force_unwrapping
   - generic_type_name
   - identical_operands
-  - identifier_name
```

Comment:
We decided to remove this since it's not enforced in other languages, and limits us from using 2-character variable names